src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -4319,7 +4324,8 @@ def _create_nw_connection_monitor_v2_test_configuration(cmd,
         ConnectionMonitorTcpConfiguration = cmd.get_models('ConnectionMonitorTcpConfiguration')
         tcp_config = ConnectionMonitorTcpConfiguration(
             port=tcp_port,
-            tcp_disable_trace_route=tcp_disable_trace_route
+            destination_port_behavior=tcp_port_behavior,
+            disable_trace_route=tcp_disable_trace_route
```

Comment:
Does if change the name of the property?

Reply:
No. It's typo last time I did here